kani-driver/src/concrete_playback.rs

```diff
@@ -251,11 +259,23 @@ fn format_unit_test(harness_name: &str, concrete_vals: &[ConcreteVal]) -> UnitTe
     let hash = hasher.finish();
 
     let concrete_playback_func_name = format!("kani_concrete_playback_{harness_name}_{hash}");
+
+    let randomize_layout_message = match randomize_layout_seed {
+        None => String::new(),
+        Some(None) => {
+            "// This test has to be run with rustc option: -Z randomize-layout\n    ".to_string()
+        }
+        Some(Some(seed)) => format!(
+            "// This test has to be run with rust options: -Z randomize-layout -Z layout-seed={}\n    ",
+            seed,
+        ),
+    };
+
     #[rustfmt::skip]
     let concrete_playback = format!(
 "#[test]
 fn {concrete_playback_func_name}() {{
-    let concrete_vals: Vec<Vec<u8>> = vec![
+    {randomize_layout_message}let concrete_vals: Vec<Vec<u8>> = vec![
```

Comment:
For code readability, this should probably be split into:
```rust
{randomize_layout_message}
let concrete_vals:
```

Reply:
The code would indeed be more readable but it would result in an empty blank line when the option isn't used
The goal was to leave the current output entirely unchanged in the most common case - no randomisation - but I don't mind adding the blank line either

Reply:
If you kept the match arms the exact same and added an escape after `randomize_layout_message`, I think it would work:
```rust
#[rustfmt::skip]
    let concrete_playback = format!(
"#[test]
fn {concrete_playback_func_name}() {{
    {randomize_layout_message}\
    let concrete_vals: Vec<Vec<u8>> = vec![
    ...
```

Reply:
Oh fair enough!